core/lib/src/route/route.rs

```diff
@@ -253,6 +255,7 @@ impl Route {
             sentinels: Vec::new(),
             handler: Box::new(handler),
             rank, uri, method,
+            doc_comment: None,
```

Comment:
```suggestion
            doc_comment: String::new(),
```

Note that `String::new()` is guaranteed not to allocate.